Resources/Prototypes/Entities/Structures/Machines/lathe.yml

```diff
@@ -319,6 +319,7 @@
       - ShellShotgun
       - ShellShotgunBeanbag
       - ShellShotgunFlare
+      - ShellTranquilizer
```

Comment:
so the one thing i'm kinda iffy on here is that for meditechfab, the tranquilizer has to be researched, while from the sec techfab, it's roundstart. This can't be fixed easily so i feel a bit conflicted about this.

Reply:
Well, tranquillizer shot can no longer be researched, as non-lethal tech was purged in #9449, where secfab received every ammo recipe by default, except tranquillizer. I think I should just remove tranquillizer from medfab menu and call it a day.